proposed/2021/VisualStudioPackageInstallDeepLink.md

```diff
@@ -0,0 +1,163 @@
+    2. Target package should be pre-selected with its package details open to the target version from NuGet.org.
+    ![Solution level PMUI Browse tab filtered to target package](.\meta/resources/VisualSutdioPackageInstallDeepLink/Filtered%20Package%20Search%20Results.jpg)
+4. User chooses relevant projects and installs the package as desired.
+
+### Scenario 2 - Visual Studio is open without a solution selected
+
+1. Deep link URL is clicked.
+2. Visual Studio solution selection window is already open, wait for solution selection.
+3. When a solution is selected, open the solution level PMUI to the Browse tab.
+    1. Search query for [PackageId: “PackageName”] is prepopulated – yielding only the target package as a result.
+    2. Target package should be pre-selected with its package details open to the target version from NuGet.org.
+4. User chooses relevant projects and installs the package as desired.
+
+### Scenario 3 - A solution is open, but the PMUI isn’t
+
+1.	Deep link URL is clicked.
+2.	Solution is already selected and open.
+3. Open the solution level PMUI to the Browse tab.
+    1. Search query for [PackageId: “PackageName”] is prepopulated – yielding only the target package as a result.
+    2. Target package should be pre-selected with its package details open to the target version from NuGet.org.
+4. User chooses relevant projects and installs the package as desired.
+
+### Scenario 4 - Solution level PMUI is open
+
+1.	Deep link URL is clicked.
+2.	Solution is already selected and open.
+3.	Solution level PMUI is already open.
+4. Go to Browse tab of the solution level PMUI.
+    1. Search query for [PackageId: “PackageName”] is prepopulated – yielding only the target package as a result.
+    2. Target package should be pre-selected with its package details open to the target version from NuGet.org.
+5. User chooses relevant projects and installs the package as desired.
+
+### Scenario 5 - Project level PMUI is open
+
+1.	Deep link URL is clicked.
+2.	Solution is already selected and open.
+3.	Project level PMUI is already open.
+4. In the PMUI window that is currently open/ in focus, open the Browse tab.
+    1. Search query for [PackageId: “PackageName”] is prepopulated – yielding only the target package as a result.
+    2. Target package should be pre-selected with its package details open to the target version from NuGet.org.
+   ![Project level PMUI Browse tab filtered to target package](./meta/resources/VisualSutdioPackageInstallDeepLink/Project%20level%20filtered%20package%20results.png)
+5. User chooses relevant projects and installs the package as desired.
+
+## Drawbacks
+
+* Older versions of Visual Studio won't support this.
+* Current proposal is limited to NuGet.org packages - generalizing to packages from any feed will increase engineering complexity significantly as we would need a custom way to isolate the target package in the Browse tab.
+
+## Prior Art
+
```

Comment:
I would add [Teams deep links](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/deep-links) as prior art. Those use teams.microsoft.com site to generate links accordingly, either for web of desktop. 

https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/deep-links